src/controller.cpp

```diff
@@ -93,8 +93,10 @@ udev_device* Controller::get_udev_device() const { return m_udev_device; }
 
 void Controller::set_active(bool v) {
   if (m_is_active != v) {
-    log_debug("activation status: " << v << " "
-                                    << m_activation_cb.target<void*>());
+    std::ostringstream oss;
+    oss << "activation status: " << v << " "
+    << m_activation_cb.target<void*>();
+    log_debug(oss.str());
```

Comment:
I did not see any errors like what you had described.  Please:

* Provide more info about your system and distro.
* Check whether issue affects f3dfdc685d65f3cbfdc34701f0ba0b434394ec3a (tagged v0.8.11)
* Attach a build log.